drivers/bbram/Kconfig.xec

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 Google Inc
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_ST_BBRAM_XEC := microchip,xec-bbram
+
+config BBRAM_XEC
+	bool "Microchip XEC Battery-backed RAM drivers"
+	depends on SOC_FAMILY_MEC
+	default $(dt_compat_enabled,${DT_COMPAT_ST_BBRAM_XEC})
```

Comment:
Not sure what the proper Kconfig rules are but through the code we're using () not {}

Reply:
Fixed in latest commit